library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1212,6 +1215,14 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
           initializationData = hevcData.first;
           nalUnitLengthFieldLength = hevcData.second;
           break;
+        case CODEC_ID_FOURCC:
+            Pair<List<byte[]>, Long> fourccData = parseFourCCPrivate(
+                new ParsableByteArray(codecPrivate));
+            if (fourccData.second != FOURCC_COMPRESSION_VC1)
```

Comment:
You could move this check directly into parseFourCCPrivate and then just have the method return the List<byte[]>.